vice/src/arch/libretro/retrostubs.c

```diff
@@ -267,7 +300,18 @@ int Core_PollEvent(int disable_physical_cursor_keys)
     }
 
     /* The check for kbt[i] here prevents the hotkey from generating C64 key events */
-    if(SHOWKEY==-1 && kbt[0] == 0 && kbt[1] == 0 && kbt[2] == 0 && kbt[3] == 0 && kbt[4] == 0)
+    /* SHOWKEY check is now in Core_Processkey to allow certain keys while SHOWKEY */
+    int processkey=1;
+    for(i = 0; i < (sizeof(kbt)/sizeof(kbt[0])); i++) {
+        if(processkey)
```

Comment:
Remove this, processkey is always 1 here.

Reply:
Hmm, over here it works fine if any kbt is pressed..? Needs of course a hotkey to be on a used c64 key.

Reply:
I just meant the if processkey thing it is fine really